src/components/Main/Main.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useCountries } from '../../hooks/useCountries.js';
+import Countries from '../Countries/Countries';
+
+import './Main.css';
+export default function Main() {
+  const { filterCountries, type, setType } = useCountries();
+
+  return (
+    <div>
+      <div className="select">
+        <select
+          value={type}
+          onChange={(e) => {
+            setType(e.target.value);
+          }}
+        >
+          <option value="all">All</option>
```

Comment:
While this works, remember when we are dealing with an API, there are times that the data can be very large, and this would make hard coding out options like this extremely difficult. The solution to this is to `map` through the data and render our options that way. This also presents an opportunity to start using array methods like `.filter` in order to return the data that you want, and the `[...new Set()]` to make sure that we do not receive any duplicate data. Let me know if you need any help figuring this out!